packages/web/src/scss/themes/index.scss

```diff
@@ -0,0 +1,25 @@
+@use 'sass:map';
+@use 'sass:string';
+@use '@themes' as themes;
+@use '../tools/themes' as themes-tools;
+
+$is-default-theme: true;
```

Comment:
If I get it correctly, this is actually a check for the first item in the loop. I think it either deserves a comment or a different variable name.